README.md

```diff
@@ -1,20 +1,122 @@
+# Azure Machine Learning Responsible AI Dashboard - Private Preview
 
-# RAI-vNext-Preview
-Welcome to the PRIVATE PREVIEW of Azure Machine Learning Responsible AI. This is an early PREVIEW, still not announced and not supported publicly.
+Welcome to the private preview for the new Responsible AI dashboard in Azure Machine Learning (AzureML) SDK and studio. The following is a guide for you to onboard to the new capabilities. For questions, please contact mithigpe@microsoft.com.
 
-# Scope for RAI in vNext versions
-- CLI (Common-Line Interface) support for Responsible AI
-- AML Python SDK vNext for Responsible AI 
+## What is this new feature?
 
-# Prerequisites
-An Azure subscription. If you don't have an Azure subscription, [create a free account](https://aka.ms/AMLFree) before you begin.
+AzureML currently supports both [model explanations](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-machine-learning-interpretability-aml) and [model fairness](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-machine-learning-fairness-aml) in public preview. As we expand our offerings under Responsible AI tools for AzureML users, this new feature brings pre-existing features and brand new offerings under one-stop-shop SDK package and studio UI dashboard:
+- Error Analysis (new): view and understand the error distributions of your model over your dataset via a decision tree map or heat map visualization.
+- Data Explorer: explore your dataset by feature sets and other metrics such as predicted Y or true Y
+- Model Statistics: explore the distribution of your model outcomes and performance metrics
+- Interpretability: view the aggregate and individual feature importances across your model and dataset
+- Counterfactual Example What-If's (new): create automatically generated diverse sets of counterfactual examples for each datapoint that is minimally perturbed in order to switch its predicted class or output. Also create your own counterfactual datapoint by perturbing feature values manually to observe the new outcome of your model prediction.
```

Comment:
"Counterfactual Example What-If's" is a bit cumbersome.

Reply:
"Counterfactual What-Ifs"